src/index.ts

```diff
@@ -0,0 +1 @@
+export { parseTemplates } from './parse-templates';
```

Comment:
I think we are still missing https://github.com/ember-template-imports/ember-template-imports/pull/37/files#r869321310 here (publishing as is, will plausibly give folks that are already using typescript an issue).

Reply:
nvm, I see https://github.com/ember-template-imports/ember-template-imports/pull/37#issuecomment-1122548666 now